framework/src/st_things/st_things.c

```diff
@@ -387,6 +392,28 @@ int st_things_start(void)
 	return ST_THINGS_ERROR_OPERATION_FAILED;
 }
 
+int st_things_push_notification_to_cloud(const char *target_uri, st_things_representation_s *resp_rep)
+{
+	THINGS_LOG_D(TAG, THINGS_FUNC_ENTRY);
+	int push_ret = 0;
+	OCRepPayload *payload = NULL;
+	payload = OCRepPayloadCreate();
```

Comment:
Better to combine this with the above statement.

Reply:
ok